### PR TITLE
57 aens pointers and 60 stable tuweni version

### LINKED ISSUE #57: support contracts and oracles as ÆNS pointers
currently we only support pointers for accounts (`account_pubkey`). we also need to support pointers for:
- oracles
- contracts

### LINKED ISSUE #60: Update tuweni to stable release version

### PR DESCRIPTION
**Pull request by marc0olo**
closes #57 
closes #60

### CODE REVIEW

File: src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/CompilerServiceTest.java [hunk 1]
```diff
@@ -89,6 +89,8 @@ public void testGenerateACI(TestContext context) throws IOException {
                 this.aeternityServiceNative.compiler.blockingGenerateACI(
                     paymentSplitterSource, null, null);
             _logger.info(aci.getEncodedAci().toString());
+            ObjectMapper mapper = new ObjectMapper();
+            System.out.println(mapper.writeValueAsString(aci.getEncodedAci()));
```
Comment (mitch-lbw) [inline src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/CompilerServiceTest.java]: remove sysout

File: src/main/java/com/kryptokrauts/aeternity/sdk/service/transaction/type/model/NameUpdateTransactionModel.java [hunk 2]
```diff
@@ -94,10 +103,32 @@ public void validateInput() {
...
+
+  private NamePointer buildNamePointer(String pointerAddress) {
+    return new NamePointer()
+        .key(identifierToPointerKeyMap.get(getIdentifier(pointerAddress)))
+        .id(pointerAddress);
+  }
+
+  private String getIdentifier(String pointerAddress) {
+    return pointerAddress.split("_")[0];
+  }
```
Comment (mitch-lbw) [inline src/main/java/com/kryptokrauts/aeternity/sdk/service/transaction/type/model/NameUpdateTransactionModel.java]: possible NPE

**Comment by marc0olo** [comment]
I think we should also validate whether given addresses are duplicate pointers. we should avoid that it is possible to provide e.g. 2 different account-addresses

File: src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/CompilerServiceTest.java (hunk 1 above)
Comment (marc0olo) [reply src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/CompilerServiceTest.java]: should be solved now

File: src/main/java/com/kryptokrauts/aeternity/sdk/service/transaction/type/model/NameUpdateTransactionModel.java (hunk 2 above)
Comment (marc0olo) [reply src/main/java/com/kryptokrauts/aeternity/sdk/service/transaction/type/model/NameUpdateTransactionModel.java]: should be solved now

**Pull request merged**